test/unit/features/options/mixins.spec.js

```diff
@@ -0,0 +1,77 @@
+import Vue from 'vue'
+const mergeOptions = Vue.util.mergeOptions
+
+describe('Mixins', () => {
```

Comment:
Let's prefix option test suites with Options. So it would be `describe('Options mixin', ...)`